grails-app/services/au/org/ala/ecodata/ProjectService.groovy

```diff
@@ -735,50 +746,49 @@ class ProjectService {
      *      And link artifacts to the project. TODO: creating project extent.
      * @return
      */
-    Integer importProjectsFromSciStarter() {
-        int ignoredProjects = 0, createdProjects = 0, updatedProjects = 0
+    @Scheduled(cron = "0 0 * * 0")
```

Comment:
ecodata has two servers - primary and reporting. This should only run on primary. At the moment this runs on both. Easy way to control it is by a configuration item.